recipes/optimade/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "optimade" %}
+{% set version = "0.12.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/optimade-{{ version }}.tar.gz
+  sha256: b7e410067ca42f75a2c5ce298d6b5f3b6e1e00a9d419bafdc3363e1b58e67100
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  entry_points:
+    - optimade-validator=optimade.validator:validate
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - email_validator ~=1.1
```

Comment:
I'm not sure conda supports the `~=` syntax. You'll have to expand that. In this case, if my PEP440 is not failing me, that should be >=1.1,<1.2.

Reply:
I had the feeling it was working during the tests, but I am also fine with just using `>=`.

Reply:
But then `pip check` complains.